src/lib/network/network_autoyast.rb

```diff
@@ -280,20 +281,34 @@ def assign_udevs_to_devs(udev_rules)
         end
         next if !matching_item
 
-        name_from = LanItems.GetDeviceName(item)
+        name_from = item_name(item)
```

Comment:
OK, I think I can even see that this is the bug fix: realizing that `name_from` may also have been renamed. Right?
So we should be able to write a test case that mimics the bug report, fails without this fix and succeeds with it.

Reply:
yes. I'd bet that I have an AY integration test for it ... but cannot find it. I'll do something with that.